src/components/FillInItem.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+class FillInItem extends React.Component {
```

Comment:
If you are going to use `React.Component` in the above line you can leave it like this: `import React from 'react'`
Or if the line above you want to leave it as it is I would change this line to: `class FillInItem extends Component {`

Reply:
OK, you right @irenegf83 I think changes are done XD